src/Components/Web/src/Forms/ModelBinding/EditContextModelBindingContextExtensions.cs

```diff
@@ -3,7 +3,7 @@
 
 using System.Globalization;
 
-namespace Microsoft.AspNetCore.Components.Forms;
+namespace Microsoft.AspNetCore.Components.Forms.ModelBinding;
```

Comment:
I would drop the namespace.

Reply:
Do you mean change it back to `Microsoft.AspNetCore.Components.Forms`?

I was trying to avoid that because then these extension methods would show up in completions for all uses of EditContext. `GetAttemptedValue` sounds like a general-purpose API that would apply on all platforms, but it only applies when "binding context extensions" was enabled.

If you want to change the namespace back, are you OK with me renaming `GetAttemptedValue` to `GetAttemptedModelBindingValue`?

Reply:
These APIs are internal, aren't they?

Reply:
Oh yes they are internal - sorry didn't notice the first name. That being the case I think neither their name nor namespace is super important so I'll let the namespace follow from its disk location. I wasn't 100% sure what other namespace you wanted so if you can clarify I'll use whatever you want, otherwise it will be `Microsoft.AspNetCore.Components.Forms.Mapping`.

Reply:
That sounds good